docs/feature_ps2_mouse.md

```diff
@@ -88,6 +88,31 @@ In your keyboard config.h:
 #endif
 ```
 
+### Interrupt Version (ARM chibios)
+
+Pretty much any two pins can be used for the (software) interrupt variant on ARM cores. The example below uses A8 for clock, and A9 for data.
+
+In rules.mk:
+
+```
+PS2_MOUSE_ENABLE = yes
+PS2_USE_INT = yes
+```
+
+In your keyboard config.h:
+
+```c
+#define PS2_CLOCK A8
+#define PS2_DATA  A9
+```
+the last define is used to select wich port DATA and CLOCK lines are wired to, and therefor the software-pin-interrupt should be configured for (replace the 'A' with your port-of-wiring-choice)
```

Comment:
This comment should be removed as its now not required.

Reply:
oh - right, done!
```suggestion
```